lib/Conversion/TritonGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -1157,7 +1315,7 @@ struct AtomicRMWOpConversion
       case RMWOp::FADD:
         rmwOp = "add";
         rmwOp += (valueElemNBits == 16 ? ".noftz" : "");
-        sTy = "f" + sBits;
+        sTy = ((isHopper && elementTy.isBF16()) ? "bf" : "f") + sBits;
```

Comment:
I don't understand the `isHopper` check here.  If it's not Hopper, shouldn't it be an *error* (maybe caught above) to try to use bf16?

Reply:
You're right. I was being a little redundant here.